src/PopChainManager/PopChainManager.spec.ts

```diff
@@ -0,0 +1,64 @@
+/// <reference path='../../typings/index.d.ts' />
+
+import popChainManager from './';
+import popEngine from '../PopEngine';
+import Pop from '../Pop';
+import Trigger from '../Trigger';
+
+describe('PopChainManager - ', () => {
+
+  describe('maybePinOrUnpinPopAndParentPops() - ', () => {
+
+    it('should add pinned-pop attribute to elements in chain', () => {
+
+      let parentEl = document.createElement('div');
+      parentEl.setAttribute('popgun', '');
+      parentEl.setAttribute('popgun-group', 'parent');
+
+      let childEl = document.createElement('div');
+      childEl.setAttribute('popgun', '');
+      childEl.setAttribute('popgun-group', 'child');
+
+      let t = new Trigger('click');
+
+      let parentPop = new Pop(parentEl, t);
+      let childPop = new Pop(childEl, t);
+
+      popEngine.addPopToPopStore('parent', parentPop);
+      popEngine.addPopToPopStore('child', childPop);
+
+      popChainManager.setParentChildRelationship(parentPop, childPop);
+      popChainManager.maybePinOrUnpinPopAndParentPops(childPop.targetEl, true);
+
+      expect(childEl.hasAttribute('pinned-pop')).toBe(true);
```

Comment:
```
 +      expect(childEl.hasAttribute('pinned-pop')).toBe(true);
 +      expect(parentEl.hasAttribute('pinned-pop')).toBe(true);
```

Can we have similar tests? For example, when we hover instead, pinned-pop should be undefined / false? Similarly Can we test if we have 3 pops in a chain, if we click the middle link, will only the parent and middle link be pinned?